app/src/main/res/layout/suggested_word.xml

```diff
@@ -17,18 +17,37 @@
   ~ along with Poet Assistant.  If not, see <http://www.gnu.org/licenses/>.
   -->
 
-<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Con: the suggested item layout now has multiple views instead of a TextView with a compound drawable.